en/topics/databases/SQL/README.md

```diff
@@ -4,11 +4,11 @@
 
 ## What is SQL? 
 
-What is a database and an RDMS? This intro article covers the basics and some flavors of SQL 
+What is a database and an RDBMS? This intro article covers the basics and some flavors of SQL 
```

Comment:
This was actually correct.

"Relational Database Management System"

Reply:
@MichaelCurrin Sorry, I thought it was more appropriate to call it with the same name as Wikipedia(RDBMS) does, to avoid misunderstanding (with RDMS and RDBMS terms, which have the same meaning).

Reply:
Oh yes, you are right for this case, Database is DB. I'll update on both languages on my side.